app/controllers/api/v1/canned_responses_controller.rb

```diff
@@ -4,28 +4,60 @@ module Api
   module V1
     # Api::V1::CannedResponseController
     class CannedResponsesController < Api::V1::ApiController
-      load_and_authorize_resource :canned_response, parent: false
+      load_and_authorize_resource :canned_response, parent: false, except: :show
+      skip_authorization_check only: :show
```

Comment:
I think we want `show` available to only admins?

Reply:
sure, ill change it